packages/react-components/react-components/src/Concepts/Icons/ReactIconGrid.tsx

```diff
@@ -8,42 +8,37 @@ import { makeStyles, shorthands } from '@griffel/react';
 
 const useStyles = makeStyles({
   grid: {
-    display: 'flex',
-    flexWrap: 'wrap',
-    justifyContent: ' flex-start',
+    display: 'grid',
+    gridTemplateColumns: 'auto auto auto auto',
```

Comment:
consider using grid minmax instead of forcing them info a 4up layout

something like `grid-template-columns: repeat(auto-fill, minmax(5rem, 1fr));`

https://developer.mozilla.org/en-US/docs/Web/CSS/minmax